chromadb/db/clickhouse.py

```diff
@@ -143,6 +143,9 @@ def create_collection(
 
         if len(dupe_check) > 0:
             if get_or_create:
+                if dupe_check[0][2] != metadata:
+                    self.update_collection(name, new_name=name, new_metadata=metadata)
```

Comment:
Do we need to update the documentation with this behavior? (get_or_create will update metadata)

Reply:
IMO this is strictly more intuitive than the current behavior of just dropping it, so I am not sure we need it in the tutorials or overview docs.

I just pushed a change to add it ito the docstrings so it will be present in generated API docs when we have those.

Reply:
Ok, I will add that before merging this.